pkg/edgexsdk/edgexsdk_test.go

```diff
@@ -207,3 +207,40 @@ func TestSetupMessageBusTrigger(t *testing.T) {
 	result := IsInstanceOf(trigger, (*messagebus.Trigger)(nil))
 	assert.True(t, result, "Expected Instance of Message Bus Trigger")
 }
+
+func TestApplicationSettings(t *testing.T) {
+	expectedSettingKey := "ApplicationName"
+	expectedSettingValue := "simple-filter-xml"
+
+	sdk := AppFunctionsSDK {}
+
+	sdk.configDir = "../../examples/simple-filter-xml/res"
+	err := sdk.initializeConfiguration()
```

Comment:
Not sure I'd prefer a unit test relying on the file system...

Reply:
No much choice here since it needs to load the toml file.

Reply:
Is this something that really needs to be unit tested? It's bootstrapping logic, not business logic. It's like unit testing a DI container. Is there going to be a mock RegistryClient added too then to cover `if sdk.useRegistry {`?

Reply:
The purpose was to validate that the new custom section in the toml file was un-marshed as expected. So useRegistry is out of scope.

Reply:
BTW, the test is for sdk.ApplicationSettings(). Calling initilizeConfiguration() is part of the test setup.

Reply:
@tsconn23 , FYI, the device-sdk-go is doing similar load from local file for their test. 
https://github.com/IOTechSystems/device-sdk-go/blob/a263cbbe41a13d47ce77296f58e97b3aa0b8b978/internal/config/loader_test.go#L113
I choose to not create a local test toml file since the one in the example has what is needed.